pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -61,13 +63,17 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 	timer := time.NewTimer(timeout)
 
 	e.reportRequiringApproval()
-	e.LogPersister.Info("Waiting for an approval...")
+
+	num := e.StageConfig.WaitApprovalStageOptions.MinApproverNum
```

Comment:
Plz also update the comment at line 55 (the head of this function) seen we change the function behavior.